src/file-events/cpp/generic_fsnotifier.cpp

```diff
@@ -236,5 +250,7 @@ Java_net_rubygrapefruit_platform_internal_jni_AbstractFileEventFunctions_invalid
 
 NativePlatformJniConstants::NativePlatformJniConstants(JavaVM* jvm)
     : JniSupport(jvm)
-    , nativeExceptionClass(getThreadEnv(), "net/rubygrapefruit/platform/NativeException") {
+    , nativeExceptionClass(getThreadEnv(), "net/rubygrapefruit/platform/NativeException")
+    , inotifyWatchesLimitTooLowExceptionClass(getThreadEnv(), "net/rubygrapefruit/platform/internal/jni/InotifyWatchesLimitTooLowException")
+    , inotifyInstanceLimitTooLowExceptionClass(getThreadEnv(), "net/rubygrapefruit/platform/internal/jni/InotifyInstanceLimitTooLowException") {
```

Comment:
These should be moved to a separate `linuxJniConstants` object.

Reply:
I moved the constants. I am not sure it is worth it. Please take a look at the commit and tell me what you think.

Reply:
It's ugly I agree, but I still think it's better this way. If you feel strongly, I'm okay with reverting, too.

Reply:
Let's leave it as it is now.